canonical-websites/snapcraft.io
# Updated python flow

## Done

Changes requested

## QA

- Pull the branch
- Visit http://0.0.0.0:8004/first-snap
- Follow the instructions and ensure everything works and looks correct
